openmdao/docs/openmdao_book/features/core_features/working_with_components/distributed_components.ipynb

```diff
@@ -37,45 +37,21 @@
    "cell_type": "markdown",
```

Comment:
<p>I would get rid of 'output' in 'A component that has at least one distributed output variable ...'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO/pull/2218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Yeah, I think I changed thoughts in the middle of the sentence. Removed.